<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cool_drinks</class>
 <widget class="QMainWindow" name="cool_drinks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cool Drinks</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_update">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>570</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F6D59F;
border: 2px solid #F8FFFE;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_category">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #F8FFFE;
color: #F8FFFE;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_topping">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>150</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #F8FFFE;
color: #F8FFFE;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_price">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>150</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #F8FFFE;
color: #F8FFFE;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_type">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FF9904;</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>570</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #E9190C;
border: 2px solid #F8FFFE;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_view">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>821</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #F8FFFE;
color: #F8FFFE;</string>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_topping">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>60</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FF9904;</string>
    </property>
    <property name="text">
     <string>Topping</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_price">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>60</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FF9904;</string>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_type">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #F8FFFE;
color: #F8FFFE;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_submit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F6D59F;
border: 2px solid #F8FFFE;
</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #288990;</string>
    </property>
    <property name="text">
     <string>Milkshakes
&amp;
Mocktails</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_category">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FF9904;</string>
    </property>
    <property name="text">
     <string>Category
Milkshake/Mocktail</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_view">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F6D59F;
border: 2px solid #F8FFFE;</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1197</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
